homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
Would it be a good idea if I wrote an issue for it, referring to the forum post, to make it clear that this issue has been fixed in the next release?